### PR TITLE
Update allure jest README

### PR DESCRIPTION
**Pull request by vovsemenv**

### CODE REVIEW

**Comment by vovsemenv** [comment]
@baev ?

**Comment by vovsemenv** [comment]
to fix questions like this https://t.me/allure_ru/30763

File: packages/allure-jest/README.md [hunk 1]
```diff
@@ -2,21 +2,29 @@
...
+> This integration don't work with default runner since jest 17+.
+> To use this integration after jest 17 you need to change default runner to `Jasmine`.
+
 # How to use
 
+You can find example setup and usage in this [repo](https://github.com/vovsemenv/allure-jest-example)
+
 ## Default test runner (Jasmine)
 
 By default jest uses jasmine as test runner, so you can configure allure-jasmine reporter for it:
```
Comment (baev) [inline packages/allure-jest/README.md:15]: now circus is used by default, so we need to add example how to change the runner, and what to do if you want to use circus

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/allure-jest/README.md (hunk 1 above)
Comment (vovsemenv) [reply packages/allure-jest/README.md]: fixed

**Review by baev — APPROVED** [review]

**Pull request merged**